src/js/unitDivisorSonificationStrategy.js

```diff
@@ -0,0 +1,257 @@
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy", {
+        // can override these to control sonification behaviour
+        // These need better explanations of what they do
+        sonificationOptions: {
+            strategies: {
+                unitDivisor: {
+                    config: {
+                        notes: {
+                            durations: {
+                                play: {
+                                    divisorReturnValue: 3 / 8,
+                                    remainderReturnValue: 1 / 8
+                                }
+                            },
+                            values: {
+                                divisorReturnValue: 91,
+                                remainderReturnValue: 89
+                            }
+                        },
+                        envelope: {
+                            durations: {
+                                play: {
+                                    divisorReturnValue: 1 / 8,
+                                    remainderReturnValue: 1 / 24
+                                },
+                                silence: {
+                                    divisorReturnValue: 1 / 4,
+                                    remainderReturnValue: 1 / 12
+                                }
+                            },
+                            values: {
+                                trueValue: 1.0,
+                                falseValue: 0.0
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        invokers: {
+            "sonifyData": {
+                funcName: "floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy.dataEntriesToSonificationData",
+                args: "{that}"
+            },
+            "unitDivisor10xSonificationStrategy": {
+                funcName: "floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy.unitDivisorStrategy",
+                args: ["{that}", 10]
+            },
+            "unitDivisor5xSonificationStrategy": {
+                funcName: "floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy.unitDivisorStrategy",
+                args: ["{that}", 5]
+            },
+            "unitDivisor1xSonificationStrategy": {
+                funcName: "floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy.unitDivisorStrategy",
+                args: ["{that}", 1]
+            }
+        }
+    });
+
+    // Given an object in the style of floe.chartAuthoring.dataEntryPanel.model.dataEntries,
+    // convert it to an array of objects in the style used by the sonification components,
+    // maintaining object constancy by using the dataEntry object name as the key
+    floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy.dataEntriesToSonificationData = function (that) {
```

Comment:
This can be done more declaratively: 
```
invokers: {
    "sonifyData": {
        changePath: "sonifiedData",
        value: "@expand:{that}.unitDivisorStrategy(10)"
    }
}
```